Fixes camera rotation in 3D pose export.

erak opened the pull request:
@nikozoe Could you please also verify that this fixes the rotation issue?

erak committed:
Fixes camera rotation in 3D pose export.

nikozoe reviewed (commented):
looks like the indent is wrong, like this it is still in the if clause.  
But I realized, that we either need the flag "viz_export" to be set in the predict_3d_pose.py script for the transformation to be applied to the data before it is saved, or we need to copy the transformation code that here is only inside the "if args.viz_output is not None:"  to inside the "if args.viz_export is not None:"

Commit:
Update run.py

fixed wrong indent and apply transformation to real world space also when flag -viz-output is not set
